src/components/todo/Header.js

```diff
@@ -0,0 +1,18 @@
+import { TodoTitle } from "./TodoTitle";
+import { TodoInput } from "./TodoInput";
+import React from "react";
+
+export class Header extends React.Component {
+    constructor(props) {
```

Comment:
constructor is not needed if you are not doing anything in there but calling super, as much as I remember